cubic-server/interface/LogsInterface.cpp

```diff
@@ -32,7 +32,7 @@ bool LogsInterface::on_timeout(){
     while (std::getline(file_stream, logs)) {
```

Comment:
You didn't updated your branch with master; this function is not up to date (l. 30 nt supposed to be here anymore)

Reply:
Let me do that now :)